<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="boxl"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_newFile"/>
    <addaction name="action_openFile"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_toHTML"/>
    <addaction name="action_toPDF"/>
   </widget>
   <widget class="QMenu" name="menu_add">
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="action_link"/>
    <addaction name="action_img"/>
   </widget>
   <widget class="QMenu" name="menu_style">
    <property name="title">
     <string>Стиль</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_add"/>
   <addaction name="menu_style"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_newFile">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/newfile.ico</normaloff>:/img/Toolbar/newfile.ico</iconset>
   </property>
   <property name="text">
    <string>Создать файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_openFile">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/openfile.ico</normaloff>:/img/Toolbar/openfile.ico</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/save.ico</normaloff>:/img/Toolbar/save.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/saveas.ico</normaloff>:/img/Toolbar/saveas.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="action_toHTML">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/tohtml.ico</normaloff>:/img/Toolbar/tohtml.ico</iconset>
   </property>
   <property name="text">
    <string>Экспорт в HTML</string>
   </property>
  </action>
  <action name="action_toPDF">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/topdf.ico</normaloff>:/img/Toolbar/topdf.ico</iconset>
   </property>
   <property name="text">
    <string>Экспорт в PDF</string>
   </property>
  </action>
  <action name="action_link">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/link.ico</normaloff>:/img/Toolbar/link.ico</iconset>
   </property>
   <property name="text">
    <string>Ссылка</string>
   </property>
  </action>
  <action name="action_img">
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/img/Toolbar/image.ico</normaloff>:/img/Toolbar/image.ico</iconset>
   </property>
   <property name="text">
    <string>Изображение</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
